internal/ebutil/libs_ppc64le.go

```diff
@@ -0,0 +1,7 @@
+//go:build ppc64le
+
+package ebutil
+
+// Based on https://github.com/NVIDIA/libnvidia-container/blob/v1.15.0/src/common.h#L44
```

Comment:
I'm hesitant to add many architectures if we don't have intention to actively support them. Can we just limit it to arm64 and amd64?

Reply:
Seconded; I don't foresee us supporting this arch in the future and the presence of this file may give the wrong impression.